react/features/transcribing/functions.ts

```diff
@@ -42,3 +44,28 @@ export function determineTranscriptionLanguage(config: IConfig) {
 
     return safeBCP47Locale;
 }
+
+/**
+ * Returns whether there is transcribing.
+ *
+ * @param {IReduxState} state - The redux state to search in.
+ * @returns {boolean}
+ */
+export function isTranscribing(state: IReduxState) {
+    return state['features/transcribing'].isTranscribing;
+}
+
+/**
+ * Checks whether the participant can start the transcription.
+ *
+ * @param {IReduxState} state - The redux state.
+ * @returns {boolean} - True if the participant can start the transcription.
+ */
+export function canAddTranscriber(state: IReduxState) {
```

Comment:
I think we could add a check here that iff the is a JWT, it needs to have the right feature set.